packages/core/src/components/Tabs/Tabs.jsx

```diff
@@ -46,7 +46,14 @@ function panelTabId(panel) {
  * @return {Boolean} Is this a TabPanel component?
  */
 function isTabPanel(child) {
-  return child != null && child.type === TabPanel;
+  // Preact doesn't support child.type, React doesn't support child.nodeName
+  return (
+    child != null &&
+    (child.type === TabPanel ||
+      (child.nodeName &&
```

Comment:
Could we check `child.nodeName === TabPanel` here for Preact?

Reply:
This expression never returns true with Preact.